src/tools/analyzer.md

```diff
@@ -9,7 +9,36 @@ The static analyzer evaluates your Dart code,
 checking for errors and warnings that are specified in the
 [Dart Language Specification](https://www.dartlang.org/docs/spec/).
 
-Many tools in the Dart ecosystem use the [analysis
+Many tools in the Dart ecosystem (including IDEs, DartPad, and dartanalyzer)
+perform static analysis on code. Most of these tools (with the exception
+of DartPad) allow you to customize static analysis. The next section
+points you to where you can learn how to customize static analysis.
+
+The following section,
+[Adding static analysis to your tool](#adding-static-analysis),
+applies only if you are writing a Dart tool that needs to analyze code.
+
+## Customizing static analysis
+
+You can customize the behavior of the analyzer or the linter
+by adding an analysis options file (`analysis_options.yaml`) to
+the package root of your project. The analysis options file allows
```

Comment:
I'd remove "package" because it could be confusing.

Also, "project" might be consistent with other docs, but I would have said "package", because Dart doesn't really define the notion of a "project".

Taken together this would turn into "the root of your package."